.github/FUNDING.yml

```diff
@@ -1,2 +0,0 @@
-# These are supported funding model platforms
```

Comment:
This file deletion has to be reverted for my repo

Reply:
Fixed.